src/derivatives.jl

```diff
@@ -1,10 +1,16 @@
 #=
 Single-point derivatives of scalar->scalar maps.
 =#
-function finite_difference_derivative(f, x::T, fdtype::Type{T1}=Val{:central},
-    returntype::Type{T2}=eltype(x), f_x::Union{Nothing,T}=nothing) where {T<:Number,T1,T2}
+function finite_difference_derivative(
+    f,
+    x::T,
+    fdtype::Type{T1}=Val{:central},
+    returntype::Type{T2}=eltype(x),
+    f_x::Union{Nothing,T}=nothing;
+    relstep=default_relstep(fdtype, T),
+    absstep=relstep) where {T<:Number,T1,T2}
```

Comment:
why does this default make sense? Does it recover the previous behavior?

Reply:
> Does it recover the previous behavior?

Yes, it's to match the current behavior.